<ui version="4.0" >
 <class>gasBkPointClass</class>
 <widget class="QDialog" name="gasBkPointClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>gasBkPoint</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Legal: CKT,ALV,VRG,MUS,FAT,FGF,DEL,+,-,*,/,=</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit" />
   </item>
   <item>
    <widget class="QLabel" name="lb_caution" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:10pt; color:#ff0000;">The condition is illegal, please check it again&lt;/span>&lt;span style=" font-size:8pt; color:#ff0000;">.&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bt_OK" >
       <property name="text" >
        <string>&amp;Set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_Cancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
